codecept.conf.js

```diff
@@ -66,26 +61,26 @@ exports.config = {
     customLocator: {
       enabled: true,
       showActual: true,
-      attribute: 'data-menu-id',
+      attribute: "data-menu-id",
     },
     autoLogin: {
       enabled: true,
       saveToFile: true,
-      inject: 'loginAs',
+      inject: "loginAs",
       users: {
-          dfQA: {
-              login: (I) => {
-                I.amOnPage('/');
-                I.setPopupCookie(true);
-                I.amOnPage('/');
-                I.waitForText('Dashboard', 10, '.ant-typography')
-              },
-              check: (I) => {
-                I.seeInCurrentUrl('/dashboard');
-              },
+        dfQA: {
+          login: (I) => {
+            I.amOnPage("/");
+            I.setPopupCookie(true);
+            I.amOnPage("/");
+            I.waitInUrl('/dashboard', 10);
```

Comment:
There are several parts using '..', but in here you are using "..". Can we make it as the common symbol @antuking